doc/operation-and-maintenance/Mongoose-metrics.md

```diff
@@ -0,0 +1,121 @@
+# MongooseIM metrics
+
+MongooseIM by default collects many metrics showing user behaviour and general system statistics. They are managed by [Feuerlabs/exometer](https://github.com/Feuerlabs/exometer). 
+Metrics are organized by XMPP hosts, meaning if MongooseIM servers host `a.com` and `b.com` metrics for specific host only can be obtained.
+There are also some global metrics common for every hosts.
+
+## Metrics description
+
+| Metric name | Type | Group | Description |
+| ----------- | ---- | ----- | ----------- |
+| nodeSessionCount | value | global, XMPP | Number of connected users on given MongooseIM's node|
+| totalSessionCount | value | global, XMPP | Total number of connected users to MongooseIM cluster|
+| uniqueSessionCount | value | global, XMPP | Number of unique users connected to MongooseIM cluster |
+| adhoc_local_commands | spiral | host, hook | Hook run when adhoc command was sent to the server's host |
+| adhoc_local_items | spiral | host, hook | Hook run when adhoc command was sent to a user|
+| anonymous_purge_hook | spiral | host, hook | Hook run after anonymous user disconnected |
+| c2s_stream_features | spiral | host, hook | Hook run to collect stream features from optional modules / extensions |
+| c2s_unauthenticated_iq | spiral | host, hook | Hook run when user sent iq before authentication |
+| disco_info | spiral | host, hook | |
```

Comment:
Need some help with disco related hooks. Who was dealing with this recently and know about these hooks?